### PR TITLE
feat(ingest/unity): Use ThreadPoolExecutor for CLL

### PR DESCRIPTION
**Pull request by asikowitz**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py [hunk 1]
```diff
@@ -45,6 +46,8 @@
 
 logger: logging.Logger = logging.getLogger(__name__)
 
+MAX_WORKERS = 100
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py:49]: this looks unused?

**Review by hsheth2 — APPROVED** [review]
broadly seems reasonable

assuming the tests are unchanged here, right

File: metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py (hunk 1 above)
Comment (asikowitz) [reply metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py:49]: Nice catch, was using this before I added the config param

**Comment by asikowitz** [comment]
We really need to add some databricks connector tests, if it's to be a high-ish priority source. There's some unit tests but their coverage is not great

**Pull request merged**